mbzirc_ign/CMakeLists.txt

```diff
@@ -29,5 +29,44 @@ install(DIRECTORY
   ${CMAKE_CURRENT_BINARY_DIR}/hooks
   DESTINATION share/${PROJECT_NAME})
 
+if(BUILD_TESTING)
+
+  # Fetch and configure GTest
+  include(FetchContent)
+  FetchContent_Declare(
+    googletest
+    URL https://github.com/google/googletest/archive/609281088cfefc76f9d0ce82e1ff6c30cc3591e5.zip
+  )
+  set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
+  FetchContent_MakeAvailable(googletest)
```

Comment:
Since we are already tied up in the `ros2`/`ament` ecosystem, does it make sense to use: https://github.com/ament/googletest

Reply:
Hmm this does make sense. However, I'm not sure how to use the package. Is there a good example somewhere I could see.

Reply:
https://github.com/osrf/mbzirc/pull/14